### PR TITLE
fix(make-messages-descriptive): make toastr messages more descriptive

### PR DESCRIPTION
**Pull request by Daymorelah**
#### What does this PR do?
Makes toastr messages more descriptive

#### Description of Task to be completed?
* Make the toastr message after a successful edit of an order item more descriptive.
* Make the toastr message after successful addition of a new menu be more descriptive.
* Make the toastr message after successful addition of a new engagement more descriptive.

#### How should this be manually tested?
* Clone the repo and `cd` into the project directory
* Run `npm install` to install all project dependencies
* Run `npm run build && npm run dev` to start the app
* Try to successfully edit an order item
* Try to successfully add a new menu
* Try to successfully add a new engagement

#### What are the relevant pivotal tracker stories?
[#166699006](https://www.pivotaltracker.com/story/show/166699006)

### CODE REVIEW

File: client/src/actions/admin/menuItemsAction.js [hunk 1]
```diff
@@ -125,6 +123,7 @@ const fetchVendorEngagementFailure = payload => ({
 export const fetchVendorEngagements = () => dispatch => axios
   .get(`/engagements/`)
   .then((response) => {
+    console.log('response for fetch eng is ==> ', response);
```
Comment (sulenchy) [inline client/src/actions/admin/menuItemsAction.js]: @Daymorelah `console.log` is meant for debugging(development). It should not go into production codebase. I suggest you remove all console.logs. And going forward, you can take note of this.

**Comment by sulenchy** [comment]
@Daymorelah Can you add at least two reviewers to your PR. Take note of this going forward.

File: client/src/actions/admin/menuItemsAction.js (hunk 1 above)
Comment (Daymorelah) [reply client/src/actions/admin/menuItemsAction.js]: I have fixed this issue.

File: client/.eslintrc.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "rules": {
       "max-len": [1, 80, 2],
       "quotes": 0,
-      "no-console": 1,
+      "no-console": 2,
```
Comment (sulenchy) [inline client/.eslintrc.json]: @Daymorelah Good Job for updating this

**Comment by sulenchy** [comment]
@Daymorelah Code climate is crying about the duplication of code. I can see that those snippets of code repeat were not introduced by you but you improve the codebase. You can abstract the snippet of code repeated as function and call the function various places. Good Job so far

**Pull request merged**